#!/usr/bin/env python
PACKAGE = "stiff_detection"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("th_dis", double_t, 0, "distance thresh", 0.65, 0.3, 0.8);
gen.add("th_height", double_t, 0, "height thresh", -0.6, -1, -0.4);
gen.add("th_tan_16", double_t, 0, "tangent thresh", 0.18, 0.1, 0.22);
gen.add("th_tan_32", double_t, 0, "tangent thresh", 0.15, 0.1, 0.22);
gen.add("th_z0", double_t, 0, "tangent thresh", 0.5, 0.3, 1);
gen.add("th_z1", double_t, 0, "tangent thresh", -0.5, -1, -0.5);


exit(gen.generate(PACKAGE, "stiff_detection", "stiff_dy_param"))

